### ISSUE TITLE
Option-Values Plugin documentation needs some work

### ISSUE THREAD

**Issue by padraiglennon** [rundeck/docs#218]
**Describe the bug**
I've been looking forward to using the new Option-Values Plugin functionality in the recent versions of rundeck. I'm at a loss however to how to get it working. I think the documentation page needs a bit more information such as:

- Where should the scripts/plugins/groovy scripts be placed? I've followed the script example on the documentation page but its not showing as an option when creating options. I've placed the script in the libext folder as I normally put other plugins. Is this correct?

- In the script example how does rundeck identify the plugin name? e.g. Foo Option Plugin. Are there additional steps missing here?

**Source page**
https://docs.rundeck.com/docs/developer/option-values-plugins.html

**Comment by padraiglennon** [comment]
Answering this one myself. I don't have permission to push a PR so here is the diff I created: 

`diff --git a/en/developer/12-option-values-plugins.md b/en/developer/12-option-values-plugins.md
index 9045e1b..7b6315f 100644
--- a/en/developer/12-option-values-plugins.md
+++ b/en/developer/12-option-values-plugins.md
@@ -117,23 +117,6 @@ rundeckPlugin(OptionValuesPlugin) {

 ## Script Plugin Type

-Script based Option Values plugins use the same approach to other script based plugins. See [Script Plugin Development](plugin-development.html#script-plugin-development)
-
-The changes required to create an Option Values plugin are:
-
-In the **plugin.yaml** file set the providers to something similar to this:
-
-```
-providers:
-    - name: foo-option-plugin
-      service: OptionValues
-      plugin-type: script
-      script-file: script_name
-
-```
-
-where **script_name** is the name of the script located in the **contents** folder.
-
 Options will be picked up from the script between the markers `==START_OPTIONS==` and `==END_OPTIONS==`.
 Echo your options in the format of `value:label`.
 The label will be the appear in the dropdown list, and the associated value will be used as the value passed`

**Comment by scottfrazier** [comment]
Much appreciated!

**Comment by gpguerra** [comment]
Hi, Im trying to create my own plugin, but I'm having some trouble to get the value part of the options.
I activated the plugin in the rundeck-config.properties (rundeck.feature.option-values-plugin.enabled=true)

and create a plugin same as the example.
 #!/usr/bin/env bash

echo "==START_OPTIONS=="
echo "opt11:First Option 11"
echo "opt12:Second Option 12"
echo "opt13:Second Option 13"
echo "==END_OPTIONS=="

But when I run my jobs, it always shows de labels, not the value. I'm using docker image version 3.1.0

**Comment by markfaine** [comment]
Agreed, this is very confusing documentation.  For example: Instead of or in addition to the placeholders here:
```
framework.plugin.OptionValues.[your_plugin_name].[property]=value
project.plugin.OptionValues.[your_plugin_name].[property]=value
```
There should be examples of actual values.  I have no way of knowing what is needed for either your_plugin_name or property.

In documentation like this I find that outlining a fully working example is the best way to get someone up to speed quickly.

I think a link to a properly configured zip file containing boilerplate examples for all of the various types of plugins would be helpful too.  Then it would just be a matter of following along in the documentation while you edit the example plugin.

**Comment by pgmgb** [comment]
I also struggled with the [Rundeck Option Values Plugins documentation](https://docs.rundeck.com/docs/developer/12-option-values-plugins.html#about). Finally, I was able to create the plugin as described in this [stack overflow post](https://stackoverflow.com/questions/72086284/cascading-remote-option-values-with-key-storage-values/72772050#72772050).
